sdk/eventhubs/azure-messaging-eventhubs-track1-perf/src/main/java/com/azure/messaging/eventhubs/perf/ReceiveEventsTest.java

```diff
@@ -69,13 +79,17 @@ public ReceiveEventsTest(EventHubsReceiveOptions options) {
         return Mono.empty();
     }
 
+    /**
+     * {@inheritDoc}
+     */
```

Comment:
Was this triggering a Checkstyle issue? Javadocs are inherited when using `@Override`

Reply:
Nah. There were a ton of javadoc issues. So I just did whichever I saw. :)